### ISSUE TITLE
Take back the GITHUB documentation

### ISSUE THREAD

**Issue by lamula21** [emilkowalski/sonner#147]
For accessibility purposes and eyes wellness. May you put the Sonner documentation back in GITHUB since the main website does not have dark theme? 

I will appreciate it. Thanks :)

**Comment by emilkowalski** [comment]
Added dark mode to docs in #150
